engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/NoConversionCurrying.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.interpreter.node.expression.builtin.error;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.UniquelyConstructibleBuiltin;
+
+import java.util.List;
+
+@BuiltinType
+public class NoConversionCurrying extends UniquelyConstructibleBuiltin {
+  @Override
+  protected String getConstructorName() {
+    return "Error";
+  }
+
+  @Override
+  protected List<String> getConstructorParamNames() {
+    return List.of("has_this", "has_that", "conversion");
+  }
+}
```

Comment:
Unless I'm missing something, I don't see the corresponding definition in `Errors/Common.enso`. Please add it, as each builtin should have a definition in Enso code. Also that's where we can define the `to_display_text` for it.

Reply:
Accidentally put in `built-distribution/...` that's why it worked for me. Fixed